app/assets/scripts/views/analytics-admin-area.js

```diff
@@ -31,7 +32,10 @@ var AnalyticsAA = React.createClass({
       <section className='page'>
         <div className='page__body aa'>
           <div className='aa-main'>
+            <div className="aa-header">
             <h1>{data.provinceName} Province</h1>
+            { completion || total ? <a className='bttn-s bttn-road-network' href={config.s3Bucket + data.provinceName + '.geojson'}>Download Roads</a> : '' }
```

Comment:
If a province has total, but none completed, then we have no data and wouldn't want to show a download button, right?